### PR TITLE
Create eviternal.json

### PR DESCRIPTION
**Pull request by EviternalOfficial**

### CODE REVIEW

File: domains/eviternal.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "EviternalOfficial",
+    "email": ""
```
Comment (MaskDuck) [inline domains/eviternal.json:4]: pls some email

**Review by MaskDuck — CHANGES_REQUESTED** [review]

File: domains/eviternal.json (hunk 1 above)
Comment (EviternalOfficial) [reply domains/eviternal.json:4]: Does discord work?
Discord: Vaibhav Kashyap#5747

Comment (MaskDuck) [reply domains/eviternal.json:4]: y

Comment (EviternalOfficial) [reply domains/eviternal.json:4]: wdym?

**Pull request closed without merging**